v3/src/v2/codap-v2-types.ts

```diff
@@ -544,10 +723,17 @@ export function isV2MapLegacyStorage(obj: unknown): obj is ICodapV2MapCurrentSto
 interface ICodapV2MapLayerBaseStorage {
   _links_: {
     context: IGuidLink<"DG.DataContextRecord">
-    hiddenCases?: any[],
+    // TODO_V2_IMPORT hiddenCases are not imported
+    // this array was passed right into MST where it is typed as a string array
+    // There are 296 instances where this is a non-empty array in cfm-shared
+    hiddenCases?: IGuidLink<"DG.Case">[],
     legendColl?: IGuidLink<"DG.Collection">,
     // We sometimes see an array of links here
     legendAttr?: IGuidLink<"DG.Attribute"> | IGuidLink<"DG.Attribute">[],
+    // TODO_V2_IMPORT tHiddenCases
+    // this occurs 523 times in cfm-shared
+    // in all cases the value is `[]`
+    tHiddenCases?: unknown[]
```

Comment:
Looks like detritus from a bug.

Reply:
I'm not sure what to do with these type of properties. 
If I remove them, then if we evaluate the cfm-shared again they will popup and waste time.
If one of these buggy documents is actually used, in some cases these unhandled props might cause problems. 

The best thing I can think of is to just leave them, remove the `TODO_V2_IMPORT`, and update the comment to indicate why they should be ignored.

A further step would be add a second set of types which are used for exporting V2. These would be the canonical v2 types. The importing types would extend them with these extra props which might be seen on import. To do that, I think we'd have to have two copies of the type hierarchy. The import schema would be generated from the import root.